atoMEC/staticKS.py

```diff
@@ -367,6 +378,19 @@ def construct_rho_unbound(orbs):
                 rho_unbound[i] = n_ub
                 N_unbound[i] = n_ub * config.sph_vol
 
+        # the thomas-fermi approximation is implementede
+        if config.unbound == "thomas_fermi":
+
+            # unbound density is not constant
+            for i in range(config.spindims):
+                for j in range(len(v_s[0, :])):
+                    prefac = (2.0 / config.spindims) * sqrt(2) / (pi ** 2)
```

Comment:
Change to `prefac = (2.0 / config.spindims) / (sqrt(2) * pi ** 2)` to match `ideal` prefactor (mistake in notes sorry)